Examples/IcmpFileTransfer/IcmpFileTransfer-pitcher.cpp

```diff
@@ -36,8 +36,8 @@ void usleep(__int64 usec)
 
 	ft.QuadPart = -(10 * usec);  // Convert to 100 nanosecond interval, negative value indicates relative time
 
-	timer = CreateWaitableTimer(NULL, TRUE, NULL);
-	SetWaitableTimer(timer, &ft, 0, NULL, NULL, 0);
+	timer = CreateWaitableTimer(nullptr, TRUE, nullptr);
+	SetWaitableTimer(timer, &ft, 0, nullptr, nullptr, 0);
```

Comment:
I don't know if it's a good idea to replace `NULL` with `nullptr` when calling Windows API. People have different opinions on this:
https://www.reddit.com/r/cpp/comments/82p4bx/when_msdn_says_null_is_it_okay_to_use_nullptr/?rdt=56558
https://devblogs.microsoft.com/oldnewthing/20180307-00/?p=98175

Reply:
But keeping `NULL` is also annoying, and the CI seems fine.

Reply:
The CI doesn't test `IcmpFileTransfer` example because it's not easy to test it in CI.
The build passes, but it doesn't mean the Windows API calls work the same with `NULL` / `0` and `nullptr`

Reply:
Generally, `nullptr` tends to work fine in most cases, but I do recommend testing it for correct behavior. 

Personally, I use the `NULL` macro when it is in the Win32 API call directly as the calls already have macros sprinkled in like `INFINITE` so it feels more consistent to me and use `nullptr` in all other cases, like assigning to an `LPxxx` variable. ¯\\\_(ツ)\_/¯

Reply:
Btw, maybe we should have a list of untested code... Also better to have a CI guard that warns when we touch the area without test...

Reply:
@tigercosmos you can see in codecov what's not covered: https://app.codecov.io/gh/seladb/PcapPlusPlus

Reply:
It's not straightforward though...

Reply:
@tigercosmos what do you suggest? Not sure it's easy to maintain a list of untested code, that's why we have codecov for... 🤔